samples/nodejs/Remix Aiven Postgres/remix/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install
```

Comment:
usually `npm ci` (if you have a lock file)

Reply:
Updated.